src/BenchmarkDotNet/Portability/Cpu/HardwareIntrinsics.cs

```diff
@@ -75,6 +74,7 @@ static IEnumerable<string> GetCurrentProcessInstructionSets(Platform platform)
                         break;
```

Comment:
[SERIALIZE](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.intrinsics.x86.x86serialize?view=net-7.0) is missing here.

Reply:
@MichalPetryka another great catch, thank you!

Reply:
What about adding it to NativeAOT, seems like it's [serialize](https://github.com/dotnet/runtime/blob/main/src/coreclr/tools/Common/JitInterface/ThunkGenerator/InstructionSetDesc.txt#L48) there.